### PR TITLE
Add stop_timeout to container

### PR DESCRIPTION
**Pull request by Rob0h**

### CODE REVIEW

File: container.go [hunk 1]
```diff
@@ -80,6 +81,10 @@ func (c *Container) UnmarshalYAML(unmarshal func(interface{}) error) error {
 		}
 	}
 
+	if c.StopTimeout == 0 {
+		c.StopTimeout = 10
```
Comment (divolgin) [inline container.go]: The default should be set in replicated so it's baked into the release and doesn't fluctuate with new releases on libyaml

File: container.go [hunk 2]
```diff
@@ -43,6 +43,7 @@ type Container struct {
 	PidMode              string                        `yaml:"pid_mode" json:"pid_mode"`
 	ShmSize              int64                         `yaml:"shm_size" json:"shm_size"`
 	Labels               []string                      `yaml:"labels" json:"labels"`
+	StopTimeout          uint                          `yaml:"stop_timeout" json:"stop_timeout"`
```
Comment (divolgin) [inline container.go]: I think this should be signed so that -1 == never timeout, 0 == current default, everything else is the timeout.

Comment (Rob0h) [reply container.go]: going to leave it as unsigned for now as we'll need to add a method for the docker client to stop containers without a timeout arg... thoughts @emosbaugh ?

**Comment by emosbaugh** [comment]
Shouldn’t this be a string so it can be templated? Otherwise it will come up as a support request.

File: container.go [hunk 3]
```diff
@@ -43,6 +43,7 @@ type Container struct {
 	PidMode              string                        `yaml:"pid_mode" json:"pid_mode"`
 	ShmSize              int64                         `yaml:"shm_size" json:"shm_size"`
 	Labels               []string                      `yaml:"labels" json:"labels"`
+	StopTimeout          string                        `yaml:"stop_timeout" json:"stop_timeout"`
```
Comment (emosbaugh) [inline container.go]: you could use a UintString here
https://github.com/replicatedhq/libyaml/blob/e8bad50650727997f11bf01d21a8817a7974fbed/uint_string.go

**Review by emosbaugh — APPROVED** [review]

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**